test/test_tamper_evident.py

```diff
@@ -73,7 +73,7 @@ def test_editing_is_detected(self):
         self.assertFalse(tef.validate())
 
     def test_oneline_file_is_detected(self):
-        filename = self.write_tamper_evident(b"Am I OK?")
+        filename = self.write_tamper_evident("Am I OK?")
```

Comment:
Have you two identified the pain points? Are there still issues with bytes vs. unicode, or is this ready for final review?